### DISCUSSION TITLE
Default currency for format string

### DISCUSSION THREAD

**Discussion by databayes** [TabularEditor/TabularEditor3#1038] [Ideas]
Is there a way to change the default currency string to £  for example, when setting the format type of currency when creating a measure in the properties pane?

**Comment by DBojsen** [comment]
Hi Sue

Currently we unfortunately hardcode it with a $ sign.
I have added a backlog item to make this configurable.

BR David - TE3 Support

**Comment by otykier** [comment]
Hi @databayes 
I'm happy to let you know, that our [latest release of TE3](https://docs.tabulareditor.com/te3/other/release-notes/3_15_0.html?tabs=downloads) now allows you to change the default currency format. Find this setting under **Tools > Preferences > TOM Explorer > Localization**:

![image](https://github.com/TabularEditor/TabularEditor3/assets/8976200/b9104124-cb1d-44cf-9b8e-3c59aa980f93)

**Comment by databayes** [comment]
Brilliant, thanks Daniel 🙂
